runtime-modules/forum/src/lib.rs

```diff
@@ -1955,9 +1973,19 @@ impl<T: Trait> Module<T> {
         // Ensure actor can moderate the category
         let category = Self::ensure_can_moderate_category(&account_id, &actor, &category_id)?;
 
+        // Helps to prevent thread ID duplicates.
+        let mut unique_stickied_ids = BTreeSet::<T::ThreadId>::new();
+
         // Ensure all thread id valid and is under the category
-        for item in stickied_ids {
-            Self::ensure_thread_exists(&category_id, item)?;
+        for thread_id in stickied_ids {
+            // Check for ID duplicates.
+            if unique_stickied_ids.contains(thread_id) {
```

Comment:
Do we need to check for duplicates? Can't we just insert them and ignore it?

Reply:
It's a fix for https://github.com/Joystream/audits/issues/6

Reply:
My bad, I missed that.